ui/src/routes/_layout/organizations/$orgId/products/$productId/repositories/$repoId/create-run.tsx

```diff
@@ -334,6 +405,22 @@ const CreateRunPage = () => {
                 'Create'
               )}
             </Button>
+            {Object.keys(form.formState.errors).length > 0 && (
+              <Card className='ml-4 bg-destructive text-sm'>
+                <CardHeader>
+                  <CardTitle>There are errors in the form</CardTitle>
+                </CardHeader>
+                <CardContent>
+                  {flattenErrors(form.formState.errors).map(
+                    ({ path, message }) => (
+                      <div key={path}>
+                        <strong>{path}:</strong> {message}
+                      </div>
+                    )
+                  )}
+                </CardContent>
+              </Card>
+            )}
```

Comment:
IMO the red card is a little too much - maybe text above or below the submit button styled like `FormMessage` would work?